data/links.json

```diff
@@ -140,6 +145,11 @@
         "icon": "fa:slack",
         "popular": true
       },
+      {
+        "name": "Swag",
+        "href": "https://swag.csh.rit.edu",
+        "icon": "material:attach-money"
```

Comment:
I think something that indicates apparel or material goods would be a better icon for this.